webpack.mix.js

```diff
@@ -44,7 +45,13 @@ mix
   // Reduce noise in Webpack output
   .webpackConfig({
     stats: "errors-only",
-    plugins: [webpackNotifier]
+    plugins: [
+      webpackNotifier,
+      new CompressionPlugin({
+        compressionOptions: { level: 9 },
+        test: /\.js$|\.css$|\.html$|\.svg$/
```

Comment:
🎉 Nice and simple. I love it! I wonder if we could only enable this in production though? Maybe not worth worrying about now, but just a thought

Reply:
So in my testing there hasn't been any noticeable performance hit, I'll definitely defer to whatever you want on this since I don't have a strong opinion.

Reply:
Let's keep it as-is. Definitely not worth slowing this down. People can tweak it if it starts to become an issue 👍